public/js/NeighbourPlot.js

```diff
@@ -0,0 +1,199 @@
+var d3 = require('d3');
+require('d3-tip')(d3);
+
+/**
+ * Draws PCA scatterplot of the data Clicking on scatterplot points will
```

Comment:
I prefer the imperative, as in Python: "Draw PCA scatterplot of the data"

Couple of newlines missing?

Finally, let's keep it general, because I want to try tSNE soon: "Draw scatterplot of the dimension-reduced data"